grifts.go

```diff
@@ -49,9 +49,8 @@ func middlewareGrift(a *App) {
 }
 
 func printMiddleware(a *App) {
-	fmt.Println(fmt.Sprintf("-> %s", a.Name))
-	fmt.Println(a.Middleware.String())
-	fmt.Println()
+	fmt.Printf("-> %s ", a.Name)
```

Comment:
Sorry for this, but this statement is missing a newline. Last change, I swear! :)